packages/styles/scss/components/overflow-menu/_overflow-menu.scss

```diff
@@ -253,6 +253,7 @@
     background-color: transparent;
     color: $text-02;
     cursor: pointer;
+    font-family: font-family('sans');
```

Comment:
does `inherit` work here since you are already including the type-style of `body-short-01`?

Reply:
``
```suggestion
    font-family: inherit;
```

Reply:
I don't think `body-short-01` includes font-family rules? might be better to be explicit here like before